metricbeat/module/docker/container/data.go

```diff
@@ -31,7 +32,26 @@ func eventMapping(cont *dc.APIContainers) common.MapStr {
 		"status": cont.Status,
 	}
 
+// Check id container have health metrics configured
+if strings.Contains(cont.Status, "(") && strings.Contains(cont.Status, ")") {
```

Comment:
Healthcheck is available since Docker 1.12, the only way to get bracket in container status is to configure healthcheck eaven in previous versions.